### PR TITLE
Login Page API-1

### PR DESCRIPTION
**Pull request by elyric-sun-asterisk-dev**
cd backend
php artisan serve

[Pre-Conditions]
None

[Expected Output]
Expected to communicate with API and login user or admin with token.

[Notes]
None

[Screenshots]
USER:
image
image
image

ADMIN:
image
image
image

### CODE REVIEW

File: backend/app/Http/Controllers/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	public function logout(Request $request){
+    if($request->user()){
+        $request->user()->currentAccessToken()->delete();
+        return response()->json([
+            'message' => 'Admin Successfully Logged out!'
+        ]);
+    }
+    return response()->json(['error' => 'Unauthenticated'], 401);
+}
```
Comment (chevyB) [inline backend/app/Http/Controllers/AdminController.php]: This should be the proper indention and spacing. Pls apply to all functions in controller
```suggestion
	public function logout(Request $request)
	{
            if($request->user()){
                $request->user()->currentAccessToken()->delete();
                return response()->json([
                    'message' => 'Admin Successfully Logged out!'
                ]);
            }
            return response()->json(['error' => 'Unauthenticated'], 401);
        }
```

File: backend/database/migrations/2023_01_15_034134_add_role_to_admins_table.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('admins', function (Blueprint $table) {
+            $table->string('role')->default('1');
+        });
+    }
```
Comment (chevyB) [inline backend/database/migrations/2023_01_15_034134_add_role_to_admins_table.php]: What is the use of role? 
Is the role naa sa ERD?
I also noticed that Admin is separated table while ERD is not? Can you explain this?

Comment (elyric-sun-asterisk-dev) [reply backend/database/migrations/2023_01_15_034134_add_role_to_admins_table.php]: I will just delete that sir, actually dili na magamit 🙇

Comment (elyric-sun-asterisk-dev) [reply backend/database/migrations/2023_01_15_034134_add_role_to_admins_table.php]: Nachanged sir while doing it and I prefer to use separated User and Admin sir kay naglibug ko unsaon pag implement. 🙇

File: backend/routes/api.php [hunk 3]
```diff
@@ -15,8 +16,24 @@
...
-    return $request->user();
+// Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
+//     return $request->user();
+// });
+
+// ADMIN ROUTE
+Route::post('admin/login', [AdminController::class, 'login']);
+
+Route::middleware(['auth:sanctum', 'abilities:admin'])->prefix('admin')->group(function () {
+	Route::get('details', [AdminController::class, 'userDetails']);
```
Comment (chevyB) [inline backend/routes/api.php]: You dont need details for admin

Comment (elyric10-dev) [reply backend/routes/api.php]: That's only for testing the token sir 🙇
I will remove it right away.

**Review by chevyB — APPROVED** [review]

**Pull request merged**